upi/azure/deploy_azarm.sh

```diff
@@ -0,0 +1,17 @@
+echo "Start Deployment"
```

Comment:
i'm not sure we need this script at all..

Reply:
Removed.